vagga.yaml

```diff
@@ -25,9 +25,9 @@ containers:
         dependencies: true
       - !Install
         - libpq
-      - !Py3Requirements requirements/main.txt
+      - !Py3Requirements requirements/production.txt
     environ:
-      DATABASE_URL: postgresql://vagga:vagga@127.0.0.1:5433/mainDB
+      DATABASE_URL: sqlite://vagga:vagga@127.0.0.1:5433/mainDB
```

Comment:
This container will not run with postgres DB. We move all setting to other container. It's not need here anymore.